jupyterhub/templates/NOTES.txt

```diff
@@ -115,12 +115,6 @@ directly instead.
 {{ "HARD DEPRECATION: proxy.pdb has renamed to proxy.chp.pdb" | fail }}
 {{- end }}
 
-{{- if include "jupyterhub.digToTrue" (list "hub.config.Authenticator.enable_auth_state" .Values) }}
-{{- if not (include "jupyterhub.digToTrue" (list "hub.config.CryptKeeper.keys" .Values)) }}
-{{ "Using Authenticator.enable_auth_state, you also need to CryptKeeper.keys to a list with at least one key in it." | fail }}
-{{- end }}
-{{- end }}
-
```

Comment:
Note that `jupyterhub.digToTrue` and `jupyterhub.digToJson` that it in turn relied on, is now no longer needed thanks to a Helm 3.5 requirement. Helm 3.5 has a version of _sprig_ (helper library) that includes `dig`.